zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -251,19 +265,27 @@ public Response logout() {
     }
     if (isKnoxSSOEnabled()) {
       KnoxJwtRealm knoxJwtRealm = getJTWRealm();
-      data.put("redirectURL", constructKnoxUrl(knoxJwtRealm, knoxJwtRealm.getLogout()));
+      StringBuilder redirectURL = new StringBuilder(knoxJwtRealm.getProviderUrl());
+      String redirect = knoxJwtRealm.getRedirectParam();
+      data.put("redirectURL", constructUrl(redirectURL, redirect,
+          knoxJwtRealm.getLogout()));
       data.put("isLogoutAPI", knoxJwtRealm.getLogoutAPI().toString());
+    } else if (isKerberosRealmEnabled()) {
+      KerberosRealm kerberosRealm = getKerberosRealm();
+      StringBuilder redirectURL = new StringBuilder(kerberosRealm.getProviderUrl());
+      String redirect = kerberosRealm.getRedirectParam();
+      data.put("redirectURL", constructUrl(redirectURL, redirect, kerberosRealm.getLogout()));
```

Comment:
If `isLogoutAPI` is implemented from the previous comment, then we will need to add the same here.